### DISCUSSION TITLE
Humor - pra deixar qualquer programador desgraçado da cabeça...

### DISCUSSION THREAD

**Discussion by didiraja** [frontendbr/forum#1326] [Dicas]
Vi isso aqui no twitter e não to sabendo lidar. Programador americano é tudo doido da cabeça. 🤣 

https://imgur.com/a/4f3XB

![](https://i.imgur.com/pgZrMVx.gif)

**Comment by victorferreira** [comment]
Usava logo um daqueles telefones de disco de antigamente...

![out 02d](https://user-images.githubusercontent.com/645805/50385283-09cebd80-06b2-11e9-8096-f54df390c723.gif)

fonte: [Rotary](https://thomaspark.co/project/rotary/)

**Comment by RatoX** [comment]
No Reddit eles fazem umas competições para quem faz a pior UI de todas: https://www.theverge.com/tldr/2017/6/9/15768800/reddit-worst-volume-sliders-ui-design
